test/browser_tests/page_components/footer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Footer() {
+  this.get = function() {
+    browser.get( '/' );
```

Comment:
Ok, that makes sense. Again, naming it appropriately would be really helpful. Something like "shared_components". Short and descriptive, avoiding vagueness like template, common, shared etc.